### PR TITLE
[12.0.0] Create URP Graphics Test Package

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 This PR creates a URP graphics test package. This is useful for two reasons:
  - we don't duplicate scripts and assets between test projects
  - the package has the graphics test framework dependency so other teams / packages can integrate test with URP by adding the URP graphics test package.

### Testing status
 - Run tests locally.

### Comments to reviewers
 - Common assets and scripts were moved to the new graphics test package.
 - Each split project references the new graphics test package.
 - I could see some issues / warning in the test projects but they didn't seem related to my changes so I plan to fix them in follow PR.

### CODE REVIEW

File: TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
         "com.unity.render-pipelines.core": "file:../../../com.unity.render-pipelines.core",
         "com.unity.render-pipelines.universal": "file:../../../com.unity.render-pipelines.universal",
         "com.unity.shadergraph": "file:../../../com.unity.shadergraph",
+		"com.unity.testing.urp": "file:../../../com.unity.testing.urp",
```
Comment (theo-pnv) [inline TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json:13]: Nitpicky/minor formatting issue: there's an extra TAB (or at least on the github UI)

Comment (theo-pnv) [reply TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json:13]: (Same for l57)

Comment (theo-pnv) [reply TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json:13]: And all the .json files :)

File: com.unity.testing.urp/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"name": "com.unity.testing.urp",
+	"displayName":"URP Graphic Tests Library",
+	"version": "1.0.0",
+	"unity": "2019.3",
+	"unityRelease": "0b4",
+	"description": "Common assets library for URP test projects.",
+	"keywords": ["qa", "test", "testing", "tests", "graphics", "HDRP"],
+	"dependencies": {
+	"com.unity.testframework.graphics": "7.8.7-preview",
```
Comment (theo-pnv) [inline com.unity.testing.urp/package.json:10]: The latest gfx test framework is 7.8.8-preview

File: com.unity.testing.urp/package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	"com.unity.render-pipelines.universal": "file:./com.unity.render-pipelines.universal"
```
Comment (theo-pnv) [inline com.unity.testing.urp/package.json:11]: Is a dot missing in the path? I'd expect the package to live in the parent folder, such as `../com.unity.render-pipelines.universal`

**Review by theo-pnv — APPROVED** [review]
I only reviewed the "SDET"/infra changes. Feel free to ask another review from anyone else with more urp code knowledge. 
A few minors changes needed. Once done, consider the PR approved on my side. Thanks

File: com.unity.testing.urp/package.json (hunk 3 above)
Comment (theo-pnv) [reply com.unity.testing.urp/package.json:11]: Edit: Looking at the testing.hdrp package.json I see only one dot too, so discard what I said

Comment (phi-lira) [reply com.unity.testing.urp/package.json:11]: The single `.` means current folder and `..` parent folder.

File: com.unity.testing.urp/package.json (hunk 2 above)
Comment (phi-lira) [reply com.unity.testing.urp/package.json:10]: Ok. I will update that, it must have been bumped after I branched off

File: TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json (hunk 1 above)
Comment (phi-lira) [reply TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json:13]: Arf, I edited these on notepad++, classic tabs vs spaces :( , fixing.

**Review by hdb-unity — APPROVED** [review]
Is deleted _Test/Runtime/Renderer2DTests.cs_ required in the new package, too? 

Otherwise, looks good (I think the upgraded renderer assets won't cause an issue with min. editor version).

**Comment by phi-lira** [comment]
@hdb-unity good catch. That was a mistake!

File: TestProjects/UniversalGraphicsTest_2D/Assets/Test/Runtime/Renderer2DTests.cs [hunk 4]
```diff
@@ -1,121 +0,0 @@
-using NUnit.Framework;
-using UnityEngine;
-using UnityEngine.Rendering.Universal;
-using UnityEngine.Rendering.Universal;
-
-[TestFixture]
-class Renderer2DTests
```
Comment (hdb-unity) [inline TestProjects/UniversalGraphicsTest_2D/Assets/Test/Runtime/Renderer2DTests.cs:7]: This is removed from _UniversalGraphicsTest_2D_ but is not added to the new package.

Is that expected?

Comment (phi-lira) [reply TestProjects/UniversalGraphicsTest_2D/Assets/Test/Runtime/Renderer2DTests.cs:7]: No, this is a mistake. Good catch!

**Pull request merged**